doc/getdns_context.3.in

```diff
@@ -134,6 +134,16 @@ will be called with a callback_type of GETDNS_CALLBACK_CANCEL.
 getdns_context_destroy() returns after all of the needed cleanup is done and
 callbacks are made.
 
+.LP
+If you are using getdns in a multi-threaded manner, you are then of course using
+the underlying OpenSSL library multi-threaded and the version of that library
+in use might have a requirements on this issue. You may need to provide one or 
+two functions to allow it to function properly. For example before you call 
+getdns_context_create() you may need to use 
+the openssl functions CRYPTO_set_id_callback and CRYPTO_set_locking_callback to set up
+asyncronous operation (the  application calls these functions once for initialisation).  
+Openssl 1.0.0 or later uses the CRYPTO_THREADID_set_callback function.
+
```

Comment:
> On 22 Jun 2016, at 12:48, wtoorop <notifications@github.com> wrote:
> 
> In doc/getdns_context.3.in <https://github.com/getdnsapi/getdns/pull/186#discussion_r68038693>:
> 
> > @@ -134,6 +134,16 @@ will be called with a callback_type of GETDNS_CALLBACK_CANCEL.
> >  getdns_context_destroy() returns after all of the needed cleanup is done and
> >  callbacks are made.
> >  
> > +.LP
> > +If you are using getdns in a multi-threaded manner, you are then of course using
> > +the underlying OpenSSL library multi-threaded and the version of that library
> > +in use might have a requirements on this issue. You may need to provide one or 
> > +two functions to allow it to function properly. For example before you call 
> > +getdns_context_create() you may need to use 
> > +the openssl functions CRYPTO_set_id_callback and CRYPTO_set_locking_callback to set up
> > +asyncronous operation (the  application calls these functions once for initialisation).  
> > +Openssl 1.0.0 or later uses the CRYPTO_THREADID_set_callback function.
> > +
> The text is good, but weren't we going to generate the manpages from the doxygen text eventually?
> Why not move this paragraph to the doxygen documentation of getdns_context_create()?
> 
I did this as a short term fix since we don’t have the doxygen generation yet. I have put a brief comment in the doxygen comment too, but that needs more work so it won’t harm to have this for now. 

Sara.